### DISCUSSION TITLE
Rich Presence Change

### DISCUSSION THREAD

**Discussion by Quentin-Kurth** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#87] [Ideas]
Good day,

I would change the "Plays" to "Listens", looks nicer and also fits much better.

Lg

**Comment by Quentin-Kurth** [comment]
https://upload.Quentin-Kurth.de/16.03.2023-dw5l6IfKUM.png

**Comment by purpl3F0x** [comment]
#35
